l1-contracts/contracts/state-transition/chain-deps/facets/Admin.sol

```diff
@@ -41,52 +42,29 @@ contract AdminFacet is Base, IAdmin {
     }
 
     /// @inheritdoc IAdmin
-    function setPendingAdmin(address _newPendingAdmin) external onlyGovernor {
-        // Save previous value into the stack to put it into the event later
-        address oldPendingAdmin = s.pendingAdmin;
-        // Change pending admin
-        s.pendingAdmin = _newPendingAdmin;
-        emit NewPendingAdmin(oldPendingAdmin, _newPendingAdmin);
-    }
-
-    /// @inheritdoc IAdmin
-    function acceptAdmin() external {
-        address pendingAdmin = s.pendingAdmin;
-        require(msg.sender == pendingAdmin, "n4"); // Only proposed by current admin address can claim the admin rights
-
-        address previousAdmin = s.admin;
-        s.admin = pendingAdmin;
-        delete s.pendingAdmin;
-
-        emit NewPendingAdmin(pendingAdmin, address(0));
-        emit NewAdmin(previousAdmin, pendingAdmin);
-    }
-
-    /// @inheritdoc IAdmin
-    function setValidator(address _validator, bool _active) external onlyGovernorOrAdmin {
+    function setValidator(address _validator, bool _active) external onlyStateTransitionManager {
```

Comment:
I don't think StateTransitionManager calls `setValidator` :(

Reply:
In the future Validator can be anything, and can be set by the chains governor. But for now we have the ValidatorTimelock, which has to be fixed. So we can remove the function completely, or keep the permissions.

I will also rename the governor in the Admin contract to admin, as it is confusing.